ios-base/Extensions/FontExtension.swift

```diff
@@ -10,9 +10,12 @@ import UIKit
 import RSFontSizes
 
 extension UIFont {
-  var bold: UIFont { return withWeight(.bold) }
-  var semibold: UIFont { return withWeight(.semibold) }
-  var light: UIFont { return withWeight(.ultraLight) }
+  static let h1Regular: UIFont = .systemFont(ofSize: 32, weight: .regular)
+  static let h2Regular: UIFont = .systemFont(ofSize: 16, weight: .regular)
+  static let h3Regular: UIFont = .systemFont(ofSize: 15, weight: .regular)
+  static let h1Medium: UIFont = .systemFont(ofSize: 32, weight: .medium)
+  static let h2Medium: UIFont = .systemFont(ofSize: 16, weight: .medium)
+  static let h3Medium: UIFont = .systemFont(ofSize: 15, weight: .medium)
```

Comment:
shall we use the `font(withName:)` helper here to use proportional sizes?

Reply:
do we know the font name for .systemFont?

Reply:
you can default the name param to an empty string and it will default to system if the font doesn't exist. Also it would be great to update that func to the EInsights version where we limit the proportional size

Reply:
thanks!